### ISSUE TITLE
Lost access to my devices, but I have username, password and pgp key. Help?

### ISSUE THREAD

**Issue by lsmoura** [keybase/keybase-issues#3616]
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

I have my password, my account, and my PGP key, but I’ve since lost access to the data created when I added my device (it was a docker machine), and I can’t reset my account. Can someone here reset it for me (or just delete the device so I can login in a new one)?

My account name is “carok”.

Thank you!
-----BEGIN PGP SIGNATURE-----
Version: Keybase OpenPGP v2.1.3
Comment: https://keybase.io/crypto

wsBcBAABCgAGBQJdrQlBAAoJENYthw7vm8N1u1oH/3gvG3d6pTqprn8VIbci+I2Z
kfSyK+amm7CGAqunftfX9bUinsGzWIC37mLyy/G8FEnPhu7aMsHPH6sgtJouO6dC
bnKtxG52ugxOf8Xho7+yvTBt+ZNalAWLmCkOrblztqEI2sMsACRX+khdbxVmfU7p
8F65VrhgYmryQov3Jh6UYC+RTvRSdNp8eMm2iezW7MP8HihgickWv7rrbi3d4iNR
aDB833D0byBImC101yG4V433s5i82Qj3FbJjBB/+j2ycPZ7zJPzPqY/bBjfrGddQ
g+QHUHwg48jyARABKAdsJmX11hulxd/vlFq8KCgXht08xwXwziLCTOkqrY7vdLw=
=PRLr
-----END PGP SIGNATURE-----

**Comment by Lucky225** [comment]
Your PGP key is *not* your keybase key. https://keybase.io/carok/devices your only key is "computer" -- if that device is blown away you have lost all access to your account and can only reset it as a new user with the same username that does *not* have access to previous chats, files, etc.

**Comment by lsmoura** [comment]
@Lucky225 Yep. I know that. That’s why I’m asking here for someone to reset it.

I’ve signed my message just to prove that I have access to my account. The “reset my account” button is disabled on the website, hence, posting my issue here.

**Comment by Lucky225** [comment]
> @Lucky225 Yep. I know that. That’s why I’m asking here for someone to reset it.
> 
> I’ve signed my message just to prove that I have access to my account. The “reset my account” button is disabled on the website, hence, posting my issue here.

Gotcha. I thought perhaps you were signing it trying to regain access to the account's contents and not just the username. I didn't know account reset was still disabled. Carry on.
